src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -123,20 +123,23 @@ func NewStartCommand(
 				conf.Redpanda.SeedServers = seedServers
 			}
 
-			kafkaAddr = stringOr(
+			kafkaAddr = stringSliceOr(
 				kafkaAddr,
-				os.Getenv("REDPANDA_KAFKA_ADDRESS"),
+				strings.Split(
+					os.Getenv("REDPANDA_KAFKA_ADDRESS"),
+					",",
+				),
 			)
-			kafkaApi, err := parseNamedAddress(
+			kafkaApi, err := parseNamedAddresses(
 				kafkaAddr,
-				config.Default().Redpanda.KafkaApi.Port,
+				config.DefaultKafkaPort,
```

Comment:
i don't follow why this is related to the commit message tbh. 

can you add some context into the commit itself to explain this.

Reply:
Sure! Basically `config.Default().Redpanda.KafkaApi[0].Port` was way longer.

Reply:
but this defaults to 9092 which may not be correct, no?

Reply:
ah i take it back! this is Default()